flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/TimeWindowTranslationTest.java

```diff
@@ -51,12 +53,48 @@
  */
 public class TimeWindowTranslationTest {
 
+	@Test
```

Comment:
True, then we can keep it but should update the method name and maybe give a javadoc that describes them.